rocky/assets/js/renameReports.js

```diff
@@ -1,35 +1,38 @@
 import { onDomReady } from "../js/imports/utils.js";
 
-onDomReady(initConfirmButtons);
+onDomReady(initResetButtons);
 
-export function initConfirmButtons(element) {
-  let root = element || document;
-  let confirm_buttons = root.querySelectorAll(".confirm-button");
+function initResetButtons() {
+  const resetButtons = document.querySelectorAll(".reset-button");
 
-  confirm_buttons.forEach((button) => initClickHandlers(button));
-}
+  resetButtons.forEach((button) => {
+    button.classList.add("hidden");
+
+    let input = button.closest("tr").querySelector(".name-input");
```

Comment:
You could consider hardening against the tr or input not existing;
```suggestion
    let input = button.closest("tr")?.querySelector(".name-input");
    if (!input) return;
```